src/components/app.scss

```
@@ -15,6 +15,7 @@ body {
   justify-content: center;
   background-color: var(--theme-primary-color);
   overflow-y: auto;
+  overflow-x: auto;
```

You could merge these two lines to make things more concise.

```suggestion
  overflow: auto;
```